web_widget_dropdown_dynamic/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Dynamic dropdown widget that supports resolving options from backend.
```

Comment:
Which field types are allowed in the widget?

Reply:
I plan to support character fields, initially. Maybe integer would be easy to support

Reply:
Include selection as well

Reply:
What is usecase of that?

Reply:
Restrict selection values according a criteria. Real example, in VAT declaration in Spain, you have 5 possible values in the result:

- To be debited in bank account
- To enter directly
- To return
- To be held in fiscal account
- To be compensated

First 2 values are selectable when the declaration is positive, and the other 2 when the declaration is negative. We need 3 fields (or at least 2) now for reflecting this + `attrs` in fields.